src/main/java/uk/gov/hmcts/divorce/notification/NotificationDispatcher.java

```diff
@@ -49,6 +49,23 @@ public void sendNOC(final ApplicantNotification applicantNotification,
         }
     }
 
+    // Need different logic for NOC notification when sending invite to citizen
+    public void sendNOCToParty(final ApplicantNotification applicantNotification,
```

Comment:
I'm not sure how best to name this so people can distinguish it from `sendNOC` in the future - do you think we should include CaseInvite in the name and then you may be able to remove the comment?
```suggestion
    public void sendNOCCaseInvite(final ApplicantNotification applicantNotification,
```

Reply:
Changes made to rename method